Workbooks/UsageMetrics/Usage - Sessions/Usage - Sessions.workbook

```diff
@@ -350,7 +350,7 @@
             "type": 1,
             "description": "Get bar Chart mode",
             "isRequired": true,
-            "query": "print iff('{UsageSplitBy}' == '<unset>' and '{isTimeRange}' == 'true', 'TimeBar', iff('{UsageSplitBy}' == '<unset>' and '{isTimeRange}' == 'false', 'ByValueBar', 'SplitBar'))",
+            "query": "let splitByValue = '{UsageSplitBy:escape}';\r\nprint iff(splitByValue == '<unset>' and '{isTimeRange}' == 'true', 'TimeBar', iff(splitByValue == '<unset>' and '{isTimeRange}' == 'false', 'ByValueBar', 'SplitBar'))",
```

Comment:
can you use criteria parameters for this instead of a query that just does print?  or no because of nested ands?

Reply:
actually 2 things: i think this won't work anywhere but english because `<unset>` would get localized

i think you can do this with criteria:
if {usagesplitby} is not empty then splitbar
else if {istimerange} is not empty then timebar
else byvaluebar

Reply:
<unset> is self defined value, not a label at here, won't localized. will consider to improve it at next pr.